.github/workflows/release.yml

```
@@ -222,40 +186,26 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
         with:
-          name: windows-build-debug-${{ steps.get_version.outputs.version }}
+          name: windows-build-eebug
           path: apps/flipper/build/windows/x64/runner/Release/flipper_rw.msix
           upload_release_asset: false
```

actionlint (1.7.4)

input "upload_release_asset" is not defined in action "actions/upload-artifact@v4". available inputs are "compression-level", "if-no-files-found", "include-hidden-files", "name", "overwrite", "path", "retention-days"

(action)